web/version.js

```diff
@@ -2,7 +2,15 @@ function createDropdown()
 {
 	// configurable values:
 	var defaultTitle = "mrtk_development"; // title in the dropdown for the root version of the docs
-	var versionArray = ["releases/2.0.0", "releases/2.1.0", "releases/2.2.0", "releases/2.3.0", "prerelease/2.4.0_stabilization"]; // list of all versions in the version folder
+
+	// list of all versions in the version folder
+	var versionArray = [
+		"releases/2.0.0",
+		"releases/2.1.0",
+		"releases/2.2.0",
+		"releases/2.3.0",
+		"releases/2.4.0"

```

Comment:
On a similar note, should we update our in-code documentation references to point to the actual release?

https://github.com/microsoft/MixedRealityToolkit-Unity/blob/31f1a04118d3a35a1c7855d0a01034bfa7a0cb77/Assets/MRTK/Core/Utilities/StandardShader/BaseMeshOutline.cs#L11

Reply:
That would likely require a second PR (to dev) that restores the in-code links.

Reply:
Yep, similar to what we did with NuGet in the past: https://github.com/microsoft/MixedRealityToolkit-Unity/pull/7278